splunk-otel-android-volley/src/test/java/com/splunk/rum/TracingHurlStackTest.java

```diff
@@ -189,15 +175,17 @@ public void connectionError() throws IOException, InterruptedException {
                 .hasSize(1)
                 .allSatisfy(e -> e.getName().equals(SemanticAttributes.EXCEPTION_EVENT_NAME));
 
-        verifyAttributes(span, url, null);
-
+        verifyAttributes(span, url, null, null);
     }
 
     @Test
-    public void reusedRequest() throws IOException, InterruptedException {
+    public void reusedRequest() throws IOException {
+
+        String firstResponseBody = "success1";
+        String secondResponseBody = "success2";
```

Comment:
WDYT about making lengths of these different? E.g. `first response`, `second response`